generators/generator-bot-template-generator/app/templates/generators/app/templates/settings/appsettings.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
We will probably need to make changes here based on @carlosscastro's changes, so might be worth to quickly check and sync when you have a moment.

Reply:
Standardized appsettings handled